doc/source/serve/configure-serve-deployment.md

```diff
@@ -22,6 +22,7 @@ You can also refer to the [API reference](../serve/api/doc/ray.serve.deployment_
 - `health_check_timeout_s` - Duration in seconds, that replicas wait for a health check method to return before considering it as failed. Defaults to 30s.
 - `graceful_shutdown_wait_loop_s` - Duration that replicas wait until there is no more work to be done before shutting down. Defaults to 2s.
 - `graceful_shutdown_timeout_s` - Duration to wait for a replica to gracefully shut down before being forcefully killed. Defaults to 20s.
+- `logging_config` - Logging Config for the deployment. See [LoggingConfig](../serve/api/doc/ray.serve.schema.LoggingConfig.rst) for details.
```

Comment:
I want to avoid people jumping around too much - can we add atleast a short description of why it exists and the kinds of things allowed in this config?

Reply:
oh, I was on the opposite trying to avoid duplicated thing :)  
So that maintain effort will be smaller. lmk wdyt.

I added some general functionality description.

Reply:
I think it should atleast be a short description - look at user config for an example